### ISSUE TITLE
[Discussion]: Move from clubhouse to waffle.io

### ISSUE THREAD

**Issue by oed** [uport-project/streams#5]
We are currently using clubhouse, it is good but is quite private and doesn't have a perfect integration with github. Instead an option is to use waffle.io instead which has more tight integation with github, and should work better for open source projects. Waffle might not be as strong in product management features, but do we need them?

cc @localredhead @bdresser

**Comment by localredhead** [comment]
<img width="1460" alt="screen shot 2018-05-09 at 12 00 40 pm" src="https://user-images.githubusercontent.com/152448/39825802-c54689c0-5380-11e8-87aa-18af6523e4d5.png">

It does have some metrics/burndown capabilities at a minimum

**Comment by localredhead** [comment]
I propose we setup an integration for 1 repo/team/project, using clubhouse naming conventions in order to maintain clubhouse too... and test it.  

@oed, maybe after experimenting quickly it will be possible to define/suggest a way to manage status appropriately.   Perhaps clubhouse can contain higher-level stories and the granular breakdown of the work is in waffle.io.   As long as the activity references the appropriate clubhouse story per the convention, it should be possible to find a light-weight process and allow the two tools to be used in parallel.  

One thing to remember about waffle, is it sits ontop of github, so we really aren't adding anything new at the end of the day... other than slight process change.  We could do what waffle does without waffle (github has built in swimlanes now)..  but it would not be as nice.

**Comment by bdresser** [comment]
thanks for raising @oed !

i'm open to whatever the development team wants, and i nominate @localredhead to be the decision maker for this discussion. 

my take: 
- clubhouse is fine but i'm not super attached to it. epics and milestones are nice, but we don't rely on them a ton, and i think we could replicate functionality with smart labelling
- it would be great to have sprints & stories more public – I have to do a lot of manual communication to outside stakeholders about the status of certain features & this would solve for that
- this could also improve the quality of detail on specific stories, especially ones that come from github issues in the first place. a new issue will trigger a new clubhouse story, but discussion is siloed on each and sometimes info is missed
- it can work for multiple repos in one board
- it could work well with this streams repo to track larger priorities (a la [Status](https://ideas.status.im/all)) 

i would also vote for either clubhouse OR waffle - not both. the value in any of these tools is having a known source of truth for projects' status, and splitting might be chaos. fine if we want to start small to test, but eventually let's make it simple.

**Comment by oed** [comment]
I agree that we should choose one and not run them in paralell @bdresser. However due to the large amount of stuff that we have in clubhouse currently makes it hard to do an atomic switch between them. Maybe we can do something similar to what you suggested @localredhead as a transition phase?
Another option could be to move over one **stream** at a time. Start experimenting with one see how it goes and then add more.

**Comment by c-castillo** [comment]
Agree with experimenting first. Keep in mind we have used 3 different tools in the last year, so we can have a "tool" burnout

**Comment by dazuck** [comment]
I tend to think collaboration tools are best if they are either super generic (trello, airtable) or very tailored to a specific use case. In-between, like Clubhouse, you lose both the flexibility of the former and the automation/clarity of the latter. So waffle may be good move..

**Comment by ajunge** [comment]
Research blossom.io

I agree that openness is the most critical feature we need for a tool
